### PR TITLE
Never know what to write in these new blog posts' commit messages

### PR DESCRIPTION
**Pull request by serverless-mom**
Here is a new blog post

### CODE REVIEW

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+</head>
+
+> "Trade isn't about goods. Trade is about information. Goods sit in the warehouse until information moves them.” <br></br>
+> C. J. Cherryh
+
+OpenTelemetry is the future of Observability, APM, Monitoring, whatever you want to call ‘the process of knowing what our software is doing.’ It’s becoming common knowledge that your time is better spent gaining experience with an open, standardized system for telemetry than closed-source or otherwise proprietary standard. This truth is so universally acknowledged that all the big players in the market have made announcements of how they’re embracing OpenTelemetry. Often these statements mention how ‘open is the future’ et cetera. But how committed are these teams to OpenTelemetry? In this series, we’ll talk about how native OpenTelemetry tools compare to APM products that have adopted OpenTelemetry only partially. In this article, we will explore how, in both New Relic and Datadog, OpenTelemetry data is a ‘second class citizen.’
+<!--truncate-->
+![Cover Image](/img/blog/2023/09/signal-28_cover.webp)
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: @serverless-mom need to update the cover pic here.

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+## Data: first and second class
+
+For over a decade APM companies have loved to add new and exotic kinds of monitoring data to their dashboards. The disconnect has often come in *where* that data appears. Back when Application Performance Monitoring was so connected to web frameworks that it was often referred to as RPM (Rails Performance Monitoring), only the performance of web requests was displayed front-and-center on the application dashboard, with custom metrics, cache metrics, database metrics, and everything else relegated to a secondary menu, or worse still entirely hidden unless you set up a custom dashboard.
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]:
```suggestion
For over a decade, APM companies have loved to add new and exotic kinds of monitoring data to their dashboards. The disconnect has often come in *where* that data appears. Back when Application Performance Monitoring was so connected to web frameworks that it was often referred to as RPM (Rails Performance Monitoring), only the performance of web requests was displayed front-and-center on the application dashboard, with custom metrics, cache metrics, database metrics, and everything else relegated to a secondary menu, or worse still entirely hidden unless you set up a custom dashboard.
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+It became standard to discuss as technical debt, the fact that a data type that was beloved by users was still relegated to a backwater in the dashboard. 
+
+Another issue is ease of data ingress: The standard APM data gathered by a SaaS company’s monitoring agent can be very very easy to report, while other types require custom configuration, settings, and tool chains that make it more brittle than the default. 
+
+The result is a system that is gently sloped. It guides the user, without them even knowing it, back to the happy path of using the proprietary monitoring information reported by Datadog and New Relic, and away from open source tools.
+
+New Relic: Where OpenTelemetry Hides
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]:
```suggestion
New Relic: Where OpenTelemetry Hides
```

Do we need to add something more here or delete it? @serverless-mom

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+## The Big Gap Between Datadog’s Marketing and their Tools
+
+Right from the installation stage… something feels amiss when trying to use OpenTelemetry with Datadog. There seem to be no actual use cases listed for OpenTelemetry. The guide on their documentation for trying OpenTelemetry reporting with a simple Python app is out of date, and there’s something kind of odd about the linked of articles from their OpenTelemetry overview:
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md:39]:
```suggestion
Right from the installation stage… something feels amiss when trying to use OpenTelemetry with Datadog. There seem to be no actual use cases listed for OpenTelemetry. The guide on their documentation for trying OpenTelemetry reporting with a simple Python app is out of date, and there’s something odd about the list of articles in their OpenTelemetry overview:
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+### Limitations: Linking Traces and Logs
+
+You cannot link traces and logs automatically with the DataDog OpenTelemetry tools. It’s necessary to manually patch your particular logging module or library with a processor that adds the aforementioned translated `TraceId` and `SpanId` as Log attributes marked `dd.trace_id` and `dd.span_id`, respectively.
+
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/09/firsclass-2.webp" alt="a screenshot of datadog traces"/>
+    <figcaption><i>This trace from manual OpenTelemetry instrumentation looks pretty good in Datadog, but despite several logging calls, nothing shows up here</i></figcaption>
+</figure>
+
+By contrast, with just the automatic instrumentation for Python in OpenTelemetry, an OpenTelemetry native tool like SigNoz can connect traces to related logs without the needed for an additional processor.
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md:75]:
```suggestion
By contrast, with just the automatic instrumentation for Python in OpenTelemetry, an OpenTelemetry native tool like SigNoz can connect traces to related logs without the need for an additional processor.
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/09/firsclass-5.webp" alt="a screenshot of datadog traces"/>
+    <figcaption><i>It's tough to prove an absence, but while some metrics are available, no span name appears in the metrics explorer.</i></figcaption>
+</figure>
+
+Our uniquely named `/rolldice` isn't listed. This is also true for proprietary Datadog traces.
+
+## OpenTelemetry in New Relic
+
+In stark contrast to Datadog, the New Relic team do a great job of documenting a happy path OpenTelemetry data. One route is clearly deprecated, the other clearly documented. And both automatic and manual instrumentation are clearly documented with an example repo that works on the first run (except for one line which, at least for me, [needed to be fixed](https://github.com/newrelic/newrelic-opentelemetry-examples/blob/e2e7b735fba05f53147587867355f5e37ae45fc3/getting-started-guides/python/Uninstrumented/app.py#L6)). I was reporting OTel data to New Relic in less than an hour, a great onboarding experience.
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]:
```suggestion
In stark contrast to Datadog, the New Relic team does a great job of documenting a happy path for OpenTelemetry data. One route is clearly deprecated, and the other clearly documented. And both automatic and manual instrumentation are clearly documented with an example repo that works on the first run (except for one line which, at least for me, [needed to be fixed](https://github.com/newrelic/newrelic-opentelemetry-examples/blob/e2e7b735fba05f53147587867355f5e37ae45fc3/getting-started-guides/python/Uninstrumented/app.py#L6)). I was reporting OTel data to New Relic in less than an hour, a great onboarding experience.
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+    <img src="/img/blog/2023/09/firsclass-6.webp" alt="a screenshot of new relic services"/>
+    <figcaption><i>At the service level, all OpenTelemetry data is separated from New Relic’s mainline data.
+</i></figcaption>
+</figure>
+
+Note that setting the same service name won’t fix the issue: application data, traces, and logs will always show up in a separate listing making a blended approach difficult.
+
+### A whole different experience
+
+Clicking into the dashboards for our two applications gives a very different experience for OpenTelemetry. Starting with the New Relic view its users are used to
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]:
```suggestion
Clicking into the dashboards for our two applications gives a very different experience for OpenTelemetry. Starting with the New Relic view, its users are used to.
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/09/firsclass-7.webp" alt="a screenshot of new relic services"/>
+    <figcaption><i>Note the web requests divided into percentiles with apdex
+</i></figcaption>
+</figure>
+
+The differences are subtle with an OpenTelemetry-monitored application, but an experienced New Relic user will know right away that something’s amiss
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md:159]:
```suggestion
The differences are subtle with an OpenTelemetry-monitored application, but an experienced New Relic user will know right away that something’s amiss.
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+    <img src="/img/blog/2023/09/firsclass-8.webp" alt="a screenshot of new relic services"/>
+    <figcaption><i>New Relic’s dashboard for an application monitored with OpenTelemetry.
+</i></figcaption>
+</figure>
+
+Despite these requests being handled by the web framework Flask, the New Relic interface doesn’t seem to recognize that this, too, is a web application.
+
+### Behind the scenes: different data
+
+New relic has a very useful feature to see the query that constructed any chart in the UI. Looking at the queries for even a very basic statistic like ‘throughput’ show the problem: even with a custom query or dashboard, it’s not possible to ‘stitch’ OpenTelemetry data together with proprietary New Relic monitoring.
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]:
```suggestion
New relic has a very useful feature to see the query behind any chart in the UI. Looking at the queries for even a very basic statistic like ‘throughput’ shows the problem: even with a custom query or dashboard, it’s not possible to ‘stitch’ OpenTelemetry data together with proprietary New Relic monitoring.
```

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
+---
+title: Is OpenTelemetry a first class citizen in your dashboard? 🤔 Comparing Datadog and New Relic against OpenTelemetry Native tools.
```
Comment (ankit01-oss) [inline blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: I think the title is too long and might get chipped while sharing on social platforms. We might be better off with a shorter title like: 

- Do Datadog and New Relic really support OpenTelemetry data?

Comment (pranay01) [reply blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: Agree, we should have a smaller more catchy title

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md (hunk 3 above)
Comment (pranay01) [reply blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: yeah, it seems like a stray line

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md (hunk 2 above)
Comment (serverless-mom) [reply blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: manually added suggestion

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md (hunk 6 above)
Comment (serverless-mom) [reply blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: slightly tweaked but incorporated

File: blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md (hunk 10 above)
Comment (serverless-mom) [reply blog/2023-09-14-is-opentelemetry-a-first-class-citizen-in-your-dashboard-a-datadog-and-newrelic-comparison.md]: I want a shorter title but I fear this revision getting us in trouble with CNCF

**Comment by serverless-mom** [comment]
Updates, including a new conclusion section and title, and now in place

**Comment by pranay01** [comment]
The conclusion section looks much better now

**Pull request merged**